src/icons/index.ts

```diff
@@ -5,13 +5,15 @@ import Home from '@/icons/Home';
 import Menu from '@/icons/Menu';
 import User from '@/icons/User';
 import Users from '@/icons/Users';
+import Trash from '@/icons/Trash';
 
 const Icons = {
   Book,
   Caret,
   Collapse,
   Home,
   Menu,
```

Comment:
T'es pas passé par l'index pour l'étoile 👀
```suggestion
import Star from '@/icons/Star';
import Trash from '@/icons/Trash';

const Icons = {
  Book,
  Caret,
  Collapse,
  Home,
  Menu,
  Star,
```